doc/ir-semantics.text

```diff
@@ -108,18 +118,30 @@ Update (partial) function, returns a S:
        = step CC[v] (P, T, H, E*, C*)
     when v := eval P H E e
 
+  step   CC[call f (v, ...)]
+         (P, T, H, E*, C*)
+       = (P, T, H, E* E', C* C)
```

Comment:
Maybe to give you an example of how I want this to work:

for

```
main:
  1: const y = call f (42)
  2: stop

Function f(x):
  1: return x
```

consider `step instruction continuation env` then

```
  step  (CC[call f (1)])  (main, 1) []
=>
  step (return x) (f, 1) [x -> 42]
=>
  step (CC[42]) (main, 1) []    =    step (const y = 42) (main, 1) []
=>
  step (stop) (main, 2) [y -> 42]
```